src/main/java/com/googleinterns/zoomtube/servlets/DeleteTranscriptServlet.java

```diff
@@ -0,0 +1,35 @@
+
```

Comment:
this line should be deleted.

Reply:
Done! I also added a file header to this file.